src/Facebook/InstantArticles/Transformer/Rules/ConfigurationSelectorRule.php

```diff
@@ -153,16 +159,93 @@ public function findAll($node, $selector)
      * @param $node
      * @return null
      */
-    public function getProperty($property_name, $node)
+    public function getProperty(string $property_name, \DOMNode $node): mixed
     {
         $value = null;
-        if (isset($this->properties[$property_name])) {
+        if (array_key_exists($property_name, $this->properties)) {
             $value = $this->properties[$property_name]->get($node);
         }
         return $value;
     }
 
-    public function getProperties()
+    /**
+     * @param $property_name
+     * @param $node
+     * @return the string
+     */
+    public function getPropertyString(string $property_name, \DOMNode $node): ?string
```

Comment:
Let's open an issue to retire the "type" parameter on the getter configuration? We're already forcing specific return types from the getter, so we could at least make it optional.

Reply:
Yes, agreed. #283